compiler/luci-interpreter/src/kernels/DepthToSpace.cpp

```diff
@@ -52,9 +43,13 @@ void DepthToSpace::configure()
   int32_t output_width = input_width * block_size;
   int32_t output_channels = input_channels / block_size / block_size;
 
-  assert(input_height == output_height / block_size);
-  assert(input_width == output_width / block_size);
-  assert(input_channels == output_channels * block_size * block_size);
+  LUCI_INTERPRETER_CHECK(input_height ==
+                         output_height /
+                             block_size); // This will not make Exception, but TFlite have also.
```

Comment:
This code may be used to check overflow. It would be ok to remove the comments.

Reply:
I got it. I will remove that comment!